docs/section10/10-4.md

```diff
@@ -0,0 +1,58 @@
+
+**Rule Assertion:** B-RMR = expected value                                           
+**Appendix G Section:** Section G3.1-10 HVAC Systems for the baseline building   
+**Appendix G Section Reference:**   
+**Data Lookup:** None   
+**Evaluation Context:** Evaluate each HeatingVentilationAirAconditioningSystem   
+
+**Applicability Checks:**   
+
+1. Applicable to baseline air-side systems 3 and 9.  
+
+**Function Call:**  
+
+1. get_baseline_system_types()
+2. get_heating_fuels_p()
+
+
+**Manual Check:** None   
+ 
+
+**Rule Logic:**  
+**Applicability Check 1:**   
+- Get baseline hvac system type dictionary: `baseline_hvac_sys_type_ids_dict_b = get_baseline_system_types(B_RMR)`
+- Get list of P_RMR heating_fuels: `heating_fuels_p = get_heating_fuels_p(P_RMR)`
+- Check if oil, other or propane are in the P_RMR: `if "PROPANE" in heating_fuels_p or "FUEL_OIL" in heating_fuels_p or "OTHER" in heating_fuels_p: oil_prop_other_p = TRUE`
+- Else: `oil_prop_other_p = FALSE`
+- For each hvac system in the B_RMR: `for hvac_b in B_RMR...HeatingVentilationAirConditioningSystem:`
+    - Get hvac system type for hvac_b: `hvac_sys_type_b = baseline_hvac_sys_type_ids_dict_b.keys()[list(baseline_hvac_sys_type_ids_dict_b.values()).index(hvac_b.id)]`
+    - Check if HVAC system is type 3 or 9: `if hvac_sys_type_b in ["SYS-3", "SYS-3a", "SYS-9"]:`
+        - Set boolean variable to FALSE: `fuels_other_b = FALSE`
+        - Set boolean variable to FALSE: `fuels_natural_gas_b = FALSE`
+        - Set boolean variable to FALSE: `fuels_electricity_b = FALSE`
+        - Set boolean variable to FALSE: `fuels_propane_b = FALSE`
+        - Set boolean variable to FALSE: `fuels_oil_b = FALSE`
+        Preheat is not applicable to baseline HVAC systems so preheat systems are not checked below (maybe not true if preheat is furnace for sys 5-8).  
+        - For each heating system in hvac_b: `heating_systems_b in hvac_b.heating_system:`
+            - Reset energy source variable: `energy_source_type_b = ""`
+            - Get the energy_source_type: `energy_source_type_b = heating_systems_b.energy_source_type`
+            - Check if the energy source is propane, if so then set boolean variable to true: `if energy_source_type_b in ["PROPANE"]:fuels_propane_b = TRUE`
+            - Check else if the energy source is natural gas, if so then set boolean variable to true: `elif energy_source_type_b in ["NATURAL_GAS"]:fuels_natural_gas_b = TRUE`
+            - Check else if the energy source is electricity, if so then set boolean variable to true: `elif energy_source_type_b in ["ELECTRICITY"]:fuels_electricity_b = TRUE`
+            - Check else if the energy source is fuel oil, if so then set boolean variable to true: `elif energy_source_type_b in ["FUEL_OIL"]:fuels_oil_b = TRUE`
+            - Else, set other boolean variable to true: `Else:fuels_other_b = TRUE`    
+                                     
+        **Rule Assertion:**  
```

Comment:
I would use the following logic:
1. If baseline heating fuel source is either electricity or natural gas, then PASS
2. If baseline heating fuel source is anything other than electricity, natural gas or propane then FAIL
3. If baseline heating fuel source is propane:
    - if natural gas is used for space or service water heating in the proposed design then FAIL
    - otherwise UNDERTERMINED

Reply:
@mkarpman's solution makes sense and seems straightforward. 
For this the first step would simply be to get all heating fuel types in the baseline. 
Question: would there ever be a scenario where a building could have more than 1 heating fuel type in the baseline?

Reply:
Also for #1, I was thinking, what if natural gas is not available at the building. Then it should not be modeled in the baseline. I agree with the remainder.